11ty/about.njk

```diff
@@ -0,0 +1,23 @@
+---
+layout: layouts/about.njk
+---
+
+<section>
+  <p class="introduction__body">A modern dictionary about us.<br>We define our words, but they don't define us.</p>
+  <div>
+    <p class="summary">
+      Self-Defined seeks to provide more inclusive, holistic, and fluid definitions to reflect the diverse perspectives of the modern world.
+    </p>
+    <p>
+      With the foundation of vocabulary, we can begin to understand lived experiences of people different than us. Words can provide us with a sense of identity and allow us to find kinship through common experiences.
+    </p>
+  </div>
+  <div class="box">
+    <h2 class="sub-headline">Ways to help</h2>
+    <ol class="list">
+      <li>Submit words and definitions or contribute to our code base through <a href="https://github.com/tatianamac/selfdefined/pulls" rel="noreferrer">pull requests</a> or <a href="https://github.com/tatianamac/selfdefined/issues" rel="noreferrer">issues</a>. Start with our <a href="https://github.com/selfdefined/web-app/blob/prod/CONTRIBUTING.md" rel="noreferrer">contributing guidelines</a> and <a href="/documentation/">documentation</a>.</li>
```

Comment:
Is this link correct? I don't think the codebase is in Tatiana's personal github anymore, is it?

Reply:
This is correct. @kkemple Do you want to fix this in this PR? It’s not strictly in the scope of it, so we can fix this separately.